### DISCUSSION TITLE
Ship binaries for mediainfo with the wheel

### DISCUSSION THREAD

**Discussion by brokedarius** [sbraz/pymediainfo#135] [Ideas]
Hi folks, I think the title says it all, instead of shipping with just the wrapper adding the mediainfo binaries as well would help have it there by default without the need to go look for mediainfo and it would make pymediainfo totally installable with just tools like pipx which is not the case for now where one has to install it if its not present in the machine.

**Comment by sbraz** [comment]
Hi,
Do you mean for Linux? In that case, there is #128 which I need to look into. Otherwise, there are already weels for macOS and Windows that contain the MediaInfo library.

**Comment by brokedarius** [comment]
Wow fastest github reply ever xD (yes for linux) thanks for letting me know.
